openmetadata-ui/src/main/resources/ui/src/components/Glossary/ImportGlossary/ImportGlossary.test.tsx

```diff
@@ -168,26 +178,28 @@ describe('Import Glossary', () => {
       await flushPromises();
     });
 
-    const successBadge = await screen.getByTestId('success-badge');
-    const cancelButton = await screen.getByTestId('cancel-button');
-    const previewButton = await screen.getByTestId('preview-button');
-    const fileName = await screen.getByTestId('file-name');
+    const importButton = await screen.findByTestId('import-button');
+
+    expect(await screen.findByTestId('import-results')).toBeInTheDocument();
+    expect(importButton).toBeInTheDocument();
+
+    await act(async () => {
+      fireEvent.click(importButton);
+    });
+
+    const successBadge = await screen.findByTestId('success-badge');
+    const previewButton = await screen.findByTestId('preview-button');
+    const fileName = await screen.findByTestId('file-name');
 
     expect(successBadge).toBeInTheDocument();
-    expect(cancelButton).toBeInTheDocument();
     expect(previewButton).toBeInTheDocument();
     expect(fileName).toHaveTextContent('glossary-terms.csv');
 
-    // preview should work
-
     await act(async () => {
       userEvent.click(previewButton);
     });
 
-    const importButton = await screen.getByTestId('import-button');
-
-    expect(await screen.getByTestId('import-results')).toBeInTheDocument();
-    expect(importButton).toBeInTheDocument();
+    expect(mockPush).toHaveBeenCalledTimes(1);
```

Comment:
same as above, We don't need to tests mockPush on every tests rather tests must be separated to verify each scenario